packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -273,7 +280,27 @@ export class CXLMarketingNavElement extends LitElement {
         );
 
         // Populate.
-        contextMenu.items = this._createContextMenuItems(menuItem.children);
+        const contextMenuItems = this._createContextMenuItems(menuItem.children);
+        contextMenu.addEventListener('opened-changed', () => {
+          contextMenu.items = contextMenuItems;
+          const listBox = document.querySelector('vaadin-context-menu-list-box');
+          setTimeout(() => {
+            const { width } = getComputedStyle(listBox);
```

Comment:
This is clever :+1: 

Curious, why do we have to be wrapped in a `setTimeout()`?

Reply:
From what I observed, without `setTimeout` the menu wasn't finished rendering and the width calculation was incorrect.

Reply:
So the primary complication is that we have to always add the "context content" menu item after initial menu open - just so we can get the initial width?

Reply:
Correct